From fe3eb3cecfd671d4601d4e032b190dfaf2b512ec Mon Sep 17 00:00:00 2001
From: Vitaly Prosyak <vitaly.prosyak@amd.com>
Date: Fri, 29 Mar 2019 12:47:10 -0500
Subject: [PATCH 2496/2940] drm/amd/display: Add 3dlut control flags

[Why & How]
The follow up change
Improve some naming for fields and structs

Signed-off-by: Vitaly Prosyak <vitaly.prosyak@amd.com>
Reviewed-by: Aric Cyr <Aric.Cyr@amd.com>
Acked-by: Bhawanpreet Lakha <Bhawanpreet.Lakha@amd.com>
Acked-by: Vitaly Prosyak <Vitaly.Prosyak@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 .../drm/amd/display/modules/inc/mod_shared.h  | 36 +++++++++----------
 1 file changed, 18 insertions(+), 18 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/modules/inc/mod_shared.h b/drivers/gpu/drm/amd/display/modules/inc/mod_shared.h
index 5a631854efda..9406d0a167ee 100644
--- a/drivers/gpu/drm/amd/display/modules/inc/mod_shared.h
+++ b/drivers/gpu/drm/amd/display/modules/inc/mod_shared.h
@@ -46,35 +46,35 @@ enum vrr_packet_type {
 };
 
 #if defined(CONFIG_DRM_AMD_DC_DCN2_0)
-union tm3dlut_internal_flags {
+union lut3d_control_flags {
 	unsigned int raw;
 	struct {
-		unsigned int dochroma_scale			:1;
-		unsigned int spec_version			:3;
-		unsigned int use_zero_display_black  :1;
-		unsigned int use_zero_source_black  :1;
-		unsigned int force_display_black		:6;
-		unsigned int apply_display_gamma	:1;
-		unsigned int exp_shaper_max			:6;
-		unsigned int unity3dlut				:1;
-		unsigned int bypass3dlut			:1;
-		unsigned int use3dlut				:1;
-		unsigned int less_than_dcip3		:1;
-		unsigned int override_lum			:1;
-		unsigned int reseved				:8;
+		unsigned int do_chroma_scale				:1;
+		unsigned int spec_version				:3;
+		unsigned int use_zero_display_black			:1;
+		unsigned int use_zero_source_black			:1;
+		unsigned int force_display_black			:6;
+		unsigned int apply_display_gamma			:1;
+		unsigned int exp_shaper_max				:6;
+		unsigned int unity_3dlut				:1;
+		unsigned int bypass_3dlut				:1;
+		unsigned int use_3dlut					:1;
+		unsigned int less_than_dcip3				:1;
+		unsigned int override_lum				:1;
+		unsigned int reseved					:8;
 	} bits;
 };
 
 enum tm_show_option_internal {
 	tm_show_option_internal_single_file		= 0,/*flags2 not in use*/
-	tm_show_option_internal_duplicate_file,/*use flags2*/
+	tm_show_option_internal_duplicate_file,		/*use flags2*/
 	tm_show_option_internal_duplicate_sidebyside/*use flags2*/
 };
 
-struct tm3dlut_settings {
+struct lut3d_settings {
 	unsigned char version;
-	union tm3dlut_internal_flags flags;
-	union tm3dlut_internal_flags flags2;
+	union lut3d_control_flags flags;
+	union lut3d_control_flags flags2;
 	enum tm_show_option_internal option;
 	unsigned int min_lum;/*multiplied by 100*/
 	unsigned int max_lum;
-- 
2.17.1

